src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -294,7 +294,7 @@
 
   "tile.blockhuttownhall.messageplacedalreadydel": "You have placed a Town Hall already. If you want to delete your colony to start a new one, click:",
   "tile.blockhuttownhall.messageplacedalreadyinfi": "You have placed a Town Hall already. If you want to delete your colony to start a new one, click [DELETE]. If you want to start a new colony but not delete your old one, click [ABANDON].",
-  "tile.blockhuttownhall.messageplacedalready": "You are too close to your existing Town Hall to place another.",
```

Comment:
Cant this also be sent if you are outside of a colony, but too close and trying to place one?

Reply:
no its only send within the colony

Reply:
kk